src/main/java/com/hackclub/hccore/HCCorePlugin.java

```diff
@@ -67,6 +70,13 @@ public void onEnable() {
     this.dataManager = new DataManager(this);
     this.protocolManager = ProtocolLibrary.getProtocolManager();
 
+    if (this.getConfig().getBoolean("settings.slack-link.enabled")) {
```

Comment:
the the get* method with the default of false
```suggestion
    if (this.getConfig().getBoolean("settings.slack-link.enabled", false)) {
```